source/sixtrack_input.f90

```diff
@@ -751,8 +750,8 @@ subroutine sixin_parseInputLineSTRU(inLine, iLine, iErr)
     return
   end if
 
-  if(nSplit > 40) then
-    write(lout,"(a)") "GEOMETRY> ERROR Structure input line cannot have more then 40 elements."
+  if(nSplit < 1 .or. nSplit > 40) then
```

Comment:
No empty lines allowed here? Comment-only lines may be pretty useful in the GEOM block...